microsoft/LSKV
# Watches

Requires a ccf installed from https://github.com/microsoft/CCF/pull/4537

To test:
1. `make run-virtual-verbose`
2. (in another terminal) `./etcdctl.sh watch a --debug`
3. (in another terminal) `./etcdctl.sh put a b`

When it works an output similar to 
```
PUT
a
b
```
should appear in the terminal running the watch command (step 2).

~Currently this does not work.~ **This is now working, with more work to do to get feature parity with etcd**
